docs/getting-started.md

```diff
@@ -31,28 +33,62 @@ Theia.
 Theia requires that Antrea v1.7.0 or later is installed in the Kubernetes
 cluster.
 
-For Antrea v1.7, please ensure the Flow Exporter feature of Antrea
-Agent is enabled in the Antrea deployment manifest:
+### Configuration
+
+Please ensure the Flow Exporter feature of Antrea Agent is enabled in the
+Antrea deployment manifest. You need to set both `featureGates.FlowExport`
+flag and `flowExporter.enable` flag to true.
+
+your `antrea-agent` ConfigMap should look like this:
 
 ```yaml
   antrea-agent.conf: |
     ...
     featureGates:
       ...
       FlowExporter: true
+    flowExporter:
+      ...
+      enable: true
 ```
 
-From Antrea v1.8, you can deploy Antrea through Helm by running the following
-commands:
+You can also deploy Antrea through Helm by running the following
+commands.
 
 ```bash
 helm repo add antrea https://charts.antrea.io
-helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true
+helm install antrea antrea/antrea -n kube-system --set featureGates.FlowExporter=true --set flowExporter.enable=true
 ```
 
 This will install the latest available version of Antrea with the Flow Exporter
-feature enabled. You can also install a specific version of Antrea (>= v1.8.0)
```

Comment:
Is this --version tag being removed from v1.13.0?

Reply:
No. I was thinking it might be confusing if we state the version here as this section is for version larger or equal to v1.13.
I've added it back.